docs/docs/environment-variables.md

```diff
@@ -4,6 +4,8 @@ title: "Environment Variables"
 
 You can easily provide environment variables to your site. Just add a `.env.development` and/or `.env.production` file in your root folder for development or production builds respectively. The environment variables are embedded during build time using Webpack's [DefinePlugin](https://webpack.js.org/plugins/define-plugin/). Because these variables are provided at build time, you will need restart your dev server or rebuild your site after changing them.
 
+To add environment variables to your ```gatsby-config.js```, you can use the NPM package [dotenv](https://www.npmjs.com/package/dotenv). Once you've installed dotenv and followed their setup instructions, you can use your environment variables in the same way as shown in the example below.
```

Comment:
You only need one backtick around inline code in markdown.

Also, let's make clearer the distinction between *browser* and *node* code. So break up the previous paragraph at "The environment variables". And preface that new paragraph with "For JavaScript loaded into the browser, the environment…"

Then for your new paragraph, preface it with "For JavaScript run in node.js e.g. in `gatsby-config.js` and `gatsby-node.js`, to add environment variables, you can use the NPM…"

Or something like that :-)